crates/viewer/re_space_view_spatial/src/mesh_loader.rs

```diff
@@ -44,12 +37,13 @@ impl LoadedMesh {
     pub fn load_asset3d_parts(
         name: String,
         media_type: &MediaType,
-        asset3d: &Asset3D,
+        bytes: &[u8],
         render_ctx: &RenderContext,
+        albedo_factor: &Option<AlbedoFactor>,
```

Comment:
Should I pass the whole Asset3D or just bytes and albedo_factor?

Reply:
doesn't matter much, but I reckon all callsites have `Asset3D` ready already? Then might as well pass the whole thing